sync/class.jetpack-sync-functions.php

```diff
@@ -206,8 +206,11 @@ public static function get_protocol_normalized_url( $callable, $new_value ) {
 		if ( ! $parsed_url ) {
 			return $new_value;
 		}
-
-		$scheme = $parsed_url['scheme'];
+		if( array_key_exists ( 'scheme' , $parsed_url) ) {
```

Comment:
Missing space after `if` and after `$parsed_url`, please fix.

Reply:
Okay, Sir